benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm

```diff
@@ -0,0 +1,181 @@
+# This parameter file reproduces a 2D benchmark for viscoelastic deformation
+# published in "Numerical modelling of magma dynamics coupled to tectonic
+# deformation of lithosphere and crust" by Keller, May and Kaus, 2013, 
+# Geophys. J. Int., v. 195,  p. 1406-1422. The model setup and results are
+# described in detail in section B.2.3 and figure B5.
+#
+# The benchmark examines bending and unbending (e.g., recovery) of a 
+# viscoelastic beam surrounded by a less dense and viscous (inelastic) fluid. 
+# Gravitational forces drive initial bending (elastic strain) of the beam for
+# 50 Kyr, which then recovers its shape over ~ 500 Kyr if gravity is turned 
+# off. The recovery is driven by the accumulated elastic stresses and thus
+# provides a basic test for the viscoelasticity implementation. 
+#
+# Compositional fields are used to track the viscoelastic stresses and material 
+# representing the beam. To improve the accuracy of tracking the beam interface,
+# a discontinuous discretization and bound preserving limiter is used for the 
+# compositional fields. Significantly, the time step is limited to 1 Kyr through
+# the "Maximum time step" parameter and a relatively high (0.5) CFL value. Using
+# a constant time step ensures the effective (viscoelastic) viscosity of the
+# viscoelastic beam and viscous fluid remains constants throughout the model run.
+#
+# As currently constructed, the model will run for 50 Kyr with a gravitational
+# acceleration of 10 m/s^2. To produce unbending of the beam after the model
+# has finished, change the parameter "End time" to 500 Kyr and set the 
+# gravitational acceleration to 0 m/s^2. A restart file is written every 10 Kyr 
+# and the parameter "Resume computation = auto" specifies that a model should  
+# restart from a checkpoint file if one is present.     
+
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 50e3
+set Use years in output instead of seconds = true
+set Resume computation                     = auto
+set Use direct solver for Stokes system    = false
+set Linear solver tolerance                = 1e-7
+set Nonlinear solver scheme                = IMPES
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 1
+set Number of cheap Stokes solver steps    = 2000
+set CFL number                             = 0.5
+set Maximum time step                      = 1e3
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = surface
+set Surface pressure                       = 0.
```

Comment:
Can you remove those parameters from the list that are set to default values and not very interesting for this problem? I am thinking of use direct solver, linear solver tolerance, nonlinear solver scheme,  nonlinear solver tolerance, timing output frequency.

Reply:
:+1: